graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/Message.java

```diff
@@ -256,6 +256,12 @@ public void addField(final String key, final Object value) {
             return;
         }
 
+        if (FIELD_TIMESTAMP.equals(key) && !(value instanceof DateTime) && !(value instanceof Date)) {
+            LOG.warn("Field \"{}\" must be of value DateTime. Make sure to apply a date converter when using extractors. Not overwriting field.",
```

Comment:
One very small thing: I'd always enclose field names etc in square brackets. This way a field name cannot mess up or confuse the meaning of the message.